### ISSUE TITLE
.Net client library error

### ISSUE THREAD

**Issue by magdev2022** [socketio/socket.io#4685]
I tried to use client library for C#. (https://github.com/doghappy/socket.io-client-csharp)
This client is only running on localhost environment.
Can solve this problem?

**Comment by OlliePugh** [comment]
A few questions:

1: This issue might be better suited for the (https://github.com/doghappy/socket.io-client-csharp) repo. As many other socket.io client libraries work perfectly find outside of localhost.

2: I am unsure of the issue you are facing, can you post some of your code or errors you are facing?

3: Or do you mean in the example on the README of the C# client repo it uses localhost, and you want clarification on how to make it publicly accessible?

TLDR; I don't understand what you are actually asking.

**Comment by magdev2022** [comment]
I solved above problem. It runs well now.
This is other question. can share simple client code with golang? (Version 4)

**Comment by OlliePugh** [comment]
If you are asking if you can use a go-lang socketio client library with a NodeJS socket.io backend, if the client library is implemented correctly, yes. But AFAIK there is no official go-lang client library, but [go-socket.io](https://github.com/googollee/go-socket.io) looks in pretty good shape.

**Comment by darrachequesne** [comment]
See also: https://github.com/socketio/socket.io/issues/3242
